File: applications/admin-web/src/app/shared/services/teacher/teacher-information-group/models/get-teacher-information-group-for-input-dto.ts
import { PagedAndSortedResultRequestDto } from "@abp/ng.core";

export class GetTeacherInformationGroupForInputDto extends PagedAndSortedResultRequestDto {
  filter: string;
  projectInformationId: string;

  constructor(initial: Partial<GetTeacherInformationGroupForInputDto> = {}) {
    super(initial);
  }
}

File: servicces/src/doan.ProjectManagement.Application.Contracts/TeacherInformationGroups/Dto/GetTeacherInformationGroupForInputDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.TeacherInformationGroups.Dto
{
    public class GetTeacherInformationGroupForInputDto : PagedAndSortedResultRequestDto
    {
        public string Filter { get; set; }
        public Guid ProjectInformationId { get; set; }
    }
}
